src/protocols/secure_channel/CASESession.cpp

```diff
@@ -123,7 +123,7 @@ using HKDF_sha_crypto = HKDF_sha;
 //
 // The session establishment fails if the response is not received within the resulting timeout window,
 // which accounts for both transport latency and the server-side latency.
-static constexpr ExchangeContext::Timeout kExpectedLowProcessingTime  = System::Clock::Seconds16(2);
+static constexpr ExchangeContext::Timeout kExpectedLowProcessingTime  = System::Clock::Seconds16(10);
```

Comment:
@vivien-apple This was introduced with https://github.com/project-chip/connectedhomeip/pull/23335 . Is it ok to raise this value for low performance targets ?

Reply:
10s seems a long time to be a default timeout that affects all platforms - this seems to be a new platform affecting all other platforms ... I believe that is not ideal

Reply:
It is indeed, on our platform it doesn't means 10 real seconds as the simulator runs very quickly when there is nothing else to do than wait on IO. In that case 2s elapses in a very short amount of time in the real world. Would it be acceptable to make it only 10s for our platform ?

Reply:
I would not expect to modify the default low processing time in core code, maybe you can consider to override this value from platform

Reply:
We can change this to a configurable value if it is okay. Then the default value will be 2s and we can override it from the platform configuration level.

Reply:
Changed to configurable value